playbooks/roles/jenkins/tasks/05_tools.yml

```diff
@@ -112,13 +119,22 @@
   tags: [apt]
   apt: pkg=nodejs=4.* state=present
 
-- name: Give jenkins user permissions on its home directory
-  file:
-    path: /var/lib/jenkins
-    owner: jenkins
-    group: jenkins
-    state: directory
-    recurse: yes
+# TODO: Stop installing PhantomJS like this when there is a better
+#       apt source available.
+- name: Download PhantomJS
+  get_url:
+    url: "https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-2.1.1-linux-x86_64.tar.bz2"
+    dest: /usr/local/src/phantomjs-2.1.1-linux-x86_64.tar.bz2
```

Comment:
It's worth checking the integrity of this downloaded binary with the md5 checksum here:
http://phantomjs.org/download.html
We've done this somewhere else but I can't find the example

Reply:
This _might_ help

https://github.com/alphagov/digitalmarketplace-jenkins/blob/master/playbooks/roles/jenkins/tasks/sops.yml#L7

Reply:
Done